hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -110,6 +110,13 @@ def is_typing_literal(thing):
     )
 
 
+def is_annotated_type(thing):
+    return (
+        type(thing).__name__ in {"AnnotatedMeta", "_AnnotatedAlias"}
```

Comment:
I'd prefer to use `isinstance` here, to avoid mistakenly applying this logic to types which merely have the same name.